akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -311,8 +311,13 @@ import scala.util.control.NonFatal
 /**
  * INTERNAL API
  */
-@InternalApi private[akka] final class QueueSink[T]()
+@InternalApi private[akka] final class QueueSink[T](maxConcurrentPulls: Int)
     extends GraphStageWithMaterializedValue[SinkShape[T], SinkQueueWithCancel[T]] {
+
+  def this() = this(1)
```

Comment:
I think this isn't used, so remove it. The delegation in the Flow dsl is a better idea anyway.

Reply:
Fixed